app/services/concerns/xccdf_report/rules.rb

```diff
@@ -19,11 +19,11 @@ def title
   end
 
   def description
-    @description ||= @rule_xml.at_css('description').text.delete!("\n")
+    @description ||= @rule_xml.at_css('description').text.delete("\n")
```

Comment:
Is this code supposed to just remove any `\n` in the text and return the result?

Reply:
Yes, exactly